package.json

```diff
@@ -55,13 +55,15 @@
     "fontAwesome": "node_modules/font-awesome-sass/assets/stylesheets"
   },
   "devDependencies": {
+    "@babel/plugin-transform-react-display-name": "^7.0.0",
```

Comment:
Use older version of plugin to make it works with older version of babel https://www.npmjs.com/package/babel-plugin-transform-react-display-name